src/dev/build/tasks/clean_tasks.js

```diff
@@ -255,3 +256,38 @@ export const CleanEmptyFoldersTask = {
     );
   },
 };
+
+export const CleanOtherPlatformFilesFromModulesTask = {
+  description: 'Cleaning files of other platforms or unnecessary sub-packages from node_modules',
+
+  async run(config, log, build) {
+    const getExtraFilesInNuclide = platform => {
+      const packageRoot = build.resolvePath('node_modules/nuclide-prebuilt-libs');
```

Comment:
we are running in wrong folder:

after `yarn build`, please check

kibana/build/default/kibana-8.0.0-SNAPSHOT-windows-x86_64/node_modules/nuclide-prebuilt-libs
and folder for other workspaces
 
I think you need to use `resolvePathForPlatform`

Reply:
Done.